docs/cli/config.md

```diff
@@ -63,4 +63,4 @@ Example (project):
 ```bash
 nio cfg project
 ```
-Will prompt you for `PK Host` and `PK Token` values to be put in the `nio.env` file of your project.
+Will prompt you for `PK Host` and `PK Token` values to be put in your projects `nio.conf` file under the [user_defined] section.
```

Comment:
project's (possesive)

and do you think `[user_defined]` because copy and paste?